sdk/test/Services/DynamoDBv2/IntegrationTests/TTLTests.cs

```diff
@@ -88,6 +88,66 @@ public void TestStoreAsEpoch()
             Assert.IsNotNull(epochMap["NonEpochDate2"].S);
         }
 
+        public void TestStoreAsAnnotatedEpoch()
```

Comment:
A few places where we duplicated tests to use with annotated classes, is there a way to parameterize the type or make it generic to avoid the duplication?

We did that for `TestHashRangeObjects` in https://github.com/aws/aws-sdk-net/commit/e69fd9481c8312887b32790414424901ae6e1e54#diff-73d1ee8f0f06d29cfa60986f66dc2fe72a23143a7390ba33cd125d66d5034d56

Reply:
We discussed internally, the new annotated types don't all descend from the base `Employee`, so it's challenging to reuse the test harness.